### PR TITLE
Feature/470019 get run details & PUT classification Id for run

### PR DESCRIPTION
**Pull request by udaydefra**

### CODE REVIEW

File: src/EPR.Calculator.API/Controllers/CalculatorController.cs [hunk 1]
```diff
@@ -172,11 +173,76 @@ public IActionResult GetCalculatorRuns([FromBody] CalculatorRunsParamsDto reques
         }
 
         [HttpGet]
-        [Route("calculatorRuns/{id}")]
-        public IActionResult GetCalculatorRun()
+        [Route("calculatorRuns/{runId}")]
+        public IActionResult GetCalculatorRun(int runId)
```
Comment (johnbenjamin-defra) [inline src/EPR.Calculator.API/Controllers/CalculatorController.cs:177]: Shall we surround with a try catch and return 500 Internal server error if any other exceptions?

File: src/EPR.Calculator.API/Controllers/CalculatorController.cs [hunk 2]
```diff
@@ -172,11 +173,76 @@ public IActionResult GetCalculatorRuns([FromBody] CalculatorRunsParamsDto reques
...
+            var calcRun = calculatorRunDetail.Run;
+            var runClassification = calculatorRunDetail.Classification;
+            var runDto = CalcRunMapper.Map(calcRun, runClassification);
+
+            return new ObjectResult(runDto);
+        }
+
+        [HttpPut]
+        [Route("calculatorRuns")]
+        public IActionResult PutCalculatorRunStatus(CalculatorRunStatusUpdateDto runStatusUpdateDto)
```
Comment (johnbenjamin-defra) [inline src/EPR.Calculator.API/Controllers/CalculatorController.cs:209]: Shall we surround with a try catch and return 500 Internal server error if any other exceptions?

**Review by johnbenjamin-defra — CHANGES_REQUESTED** [review]
Some minor changes requested.

File: src/EPR.Calculator.API/Controllers/CalculatorController.cs (hunk 1 above)
Comment (udaydefra) [reply src/EPR.Calculator.API/Controllers/CalculatorController.cs:177]: Yeah agree, Done the changes now. Thanks

File: src/EPR.Calculator.API/Controllers/CalculatorController.cs (hunk 2 above)
Comment (udaydefra) [reply src/EPR.Calculator.API/Controllers/CalculatorController.cs:214]: Yeah agree, Done the changes now. Thanks

**Review by johnbenjamin-defra — APPROVED** [review]

**Review by Mazar-Shaik — APPROVED** [review]

**Pull request merged**